dockers/chart-releaser/main.go

```diff
@@ -131,13 +132,13 @@ func main() {
 			return nil
 		}
 		log.Printf("Start uploading package %s...", info.Name())
-		return storeFile(ctx, client, path)
+		return storeFile(ctx, client, path, false)
```

Comment:
nit, add note represents `we do not disable cache in this case since ...` help the future debugging process 👍

Reply:
Sure.

Reply:
Added the comment.